core/injector/application_injector.hpp

```diff
@@ -57,6 +58,9 @@ namespace kagome::injector {
     std::shared_ptr<network::Router> injectRouter();
     std::shared_ptr<network::PeerManager> injectPeerManager();
     std::shared_ptr<api::ApiService> injectRpcApiService();
+    std::shared_ptr<network::SyncProtocolObserver> injectSyncObserver();
```

Comment:
I take it that when the PR that merges our binaries into one will be ready, there will be one injector, either?

Reply:
Sure